src/commands/database/redis/shell.tsx

```diff
@@ -8,12 +8,16 @@ import * as cp from "child_process";
 import { Text } from "ink";
 import { getConnectionDetails } from "../../../lib/database/redis/connect.js";
 import { redisArgs, withRedisId } from "../../../lib/database/redis/flags.js";
+import { sshUsageDocumentation } from "../../../lib/ssh/doc.js";
 
 export class Shell extends ExecRenderBaseCommand<
   typeof Shell,
   Record<string, never>
 > {
   static summary = "Connect to a Redis database via the redis-cli";
+  static description =
+    "This command opens an interactive redis-cli shell to a Redis database.\n\n" +
+    sshUsageDocumentation;
```

Comment:
should the new option be supported for this command as well? 

Currently the flag is not listed in the updated readme.

Reply:
Good catch. Fixed. 👌